pluma/pluma-prefs-manager-app.c

```diff
@@ -116,6 +116,22 @@ static void pluma_prefs_manager_lockdown_changed	(GSettings *settings,
 							 gchar       *key,
 							 gpointer     user_data);
 
+static void pluma_prefs_manager_draw_spaces_changed	(GSettings *settings,
+							 gchar       *key,
+							 gpointer     user_data);
+
+static void pluma_prefs_manager_draw_tabs_changed	(GSettings *settings,
+							 gchar       *key,
+							 gpointer     user_data);
+
+static void pluma_prefs_manager_draw_newlines_changed	(GSettings *settings,
+							 gchar       *key,
+							 gpointer     user_data);
+
+static void pluma_prefs_manager_draw_nbsp_changed	(GSettings *settings,
+							 gchar       *key,
+							 gpointer     user_data);
+
```

Comment:
/nit
You should align these:
```
(GSettings *settings;
 gchar     *key;
 gpointer  *user_data);
```
Seems like we need this commit so that we can see our own tabs/spaces :rofl:

Reply:
Generally I prefer to be aligned to the style used in the code. It seems strange but all the previous declaration have the same issue: eg few lines above I see
```
static void pluma_prefs_manager_lockdown_changed	(GSettings *settings,
							 gchar       *key,
							 gpointer     user_data);
```
This is the reasons why I used this strange indentation

Reply:
okay yeah, I see it now. Oh well /shrug